server/src/cm_job_task.cpp

```diff
@@ -6932,6 +6945,21 @@ ts_get_tran_info (nvplist *req, nvplist *res, char *_dbmt_error)
 
   fclose (infile);
 
+  if ((num_queries = get_sql_info (tmpfile, &query_file_size)) > 0)
+    {
+      query_p = (char *) calloc (1, query_file_size); //kshan
+      sql_info = (TS_SQL_INFO *) calloc (num_queries, sizeof (TS_SQL_INFO));
+    }
+
+  retval = get_sql_text (tmpfile, query_p, sql_info, query_file_size);	//kshan
```

Comment:
If num_queries is 0, I think get_sql_text () does not need to be called.
and It looks like the "kshan" comment should be remove.

Reply:
Thank you for your comment, comment accepted.